packages/ubuntu_init/lib/src/ubuntu_pro/ubuntu_pro_onboarding_page.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_html/flutter_html.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:ubuntu_init/src/init_step.dart';
+import 'package:ubuntu_init/ubuntu_init.dart';
+import 'package:ubuntu_provision/ubuntu_provision.dart';
+import 'package:ubuntu_utils/ubuntu_utils.dart';
+import 'package:ubuntu_wizard/ubuntu_wizard.dart';
+import 'package:yaru/yaru.dart';
+
+class UbuntuProOnboardingPage extends ConsumerWidget with ProvisioningPage {
+  const UbuntuProOnboardingPage({super.key});
+
+  @override
+  Future<bool> load(BuildContext context, WidgetRef ref) {
+    return ref.read(ubuntuProOnboardingModelProvider).init().then((_) => true);
+  }
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final l10n = UbuntuProLocalizations.of(context);
+    final model = ref.watch(ubuntuProOnboardingModelProvider);
+    return HorizontalPage(
+      windowTitle: l10n.ubuntuProPageTitle,
+      title: '',
+      bottomBar: WizardBar(
+        leading: const BackWizardButton(),
+        trailing: [
+          WizardButton(
+            label: UbuntuLocalizations.of(context).nextLabel,
+            onActivated:
+                model.selection == UbuntuProOnboardingPageSelection.skipForNow
+                    ? () => Wizard.of(context).jump(InitStep.privacy.route)
```

Comment:
Could you leave a todo note here? I'd like to get rid of this explicit jump, since it makes assumptions about the following pages. Instead we should handle this in the `load` methods of the `UbuntuProPage` and `UbuntuProSuccessPage`, by checking for some sort of 'skip' property.

```suggestion
                    ? () => Wizard.of(context).jump(InitStep.privacy.route)
                    // TODO: Handle page skipping in load methods
```

Reply:
This is actually why the `init_wizard_test` currently gets stuck [here](https://github.com/canonical/ubuntu-desktop-provision/blob/9d659cf1fd0be48968ed374bc07c9443255c933f/packages/ubuntu_init/test/init_wizard_test.dart#L226). It leaves the option button on the `UbuntuProOnboardingPage` unchanged on "Skip", so that the wizard tries to advance to the `PrivacyPage` when pressing "Next". But the privacy page isn't included in that test, so it gets stuck.